recipes/Media/channels.csv

```diff
@@ -0,0 +1,1259 @@
+L,100

```

Comment:
These files are quite unusual.  We should discuss about them again. Where do they come from?

Reply:
I processed some xml files to have most of the lines of these csv files. For like 100 to 200 of them, I listened to every record to add the missing information manually.